src/client/datascience/notebook/notebookEditorProvider.ts

```diff
@@ -128,8 +131,31 @@ export class NotebookEditorProvider implements INotebookEditorProvider {
     }
     @captureTelemetry(Telemetry.CreateNewNotebook, undefined, false)
     public async createNew(options?: { contents?: string; defaultCellLanguage: string }): Promise<INotebookEditor> {
-        const model = await this.storage.createNew(options, true);
-        return this.open(model.file);
+        const nbJson: nbformat.INotebookContent = {
+            cells: [],
+            metadata: { orig_nbformat: defaultNotebookFormat.major },
```

Comment:
Kinda strange that orig_nbformat is not optional (since it's for conversion) but I see it as non optional in the jupyter lab code, so I guess we need it.